### PR TITLE
Add missing annotations in org.mozilla.jss.ssl

### PR DESCRIPTION
**Pull request by ckelleyRH**
Resolves #705.

### CODE REVIEW

File: src/main/java/org/mozilla/jss/ssl/SSLClient.java [hunk 1]
```diff
@@ -439,7 +439,8 @@ public static void main(String argv[])  throws Exception
     this.sc = sc;
   }
 
-  public void handshakeCompleted(SSLHandshakeCompletedEvent event) {
+  @Override
+public void handshakeCompleted(SSLHandshakeCompletedEvent event) {
```
Comment (cipherboy) [inline src/main/java/org/mozilla/jss/ssl/SSLClient.java:443]: indentation? :)

File: src/main/java/org/mozilla/jss/ssl/TestCertApprovalCallback.java [hunk 2]
```diff
@@ -16,7 +16,8 @@
 public class TestCertApprovalCallback
    implements SSLCertificateApprovalCallback {
 
-	public boolean approve(
+	@Override
```
Comment (cipherboy) [inline src/main/java/org/mozilla/jss/ssl/TestCertApprovalCallback.java:19]: I think you're mixing tabs and spaces :/

**Comment by ckelleyRH** [comment]
Sigh. I pushed a second commit on top of the annotations change which is just Eclipse running auto-format on the files I changed.

**Review by edewata — APPROVED** [review]
LGTM. The bug found by Sonar seems to be a false positive, so feel free to ignore it.

**Comment by ckelleyRH** [comment]
I agree that the "bug" is not a bug, as we do eventually do `s1.equals(s2)` for value comparison rather than object comparison.

I think having both `s1.equals(s2)` and `s1 ==s2` is confusing though, it forces the dev to sit and think when all the method is intending is returning `okay` if `s1.equals(s2)` and `failed` otherwise.

I'll remove it in a further PR.

**Pull request merged**